src/mame/juku/juku.cpp

```diff
@@ -512,17 +657,23 @@ void juku_state::juku(machine_config &config)
 	m_sio[1]->rxrdy_handler().set("pic", FUNC(pic8259_device::ir0_w));
 	m_sio[1]->txrdy_handler().set("pic", FUNC(pic8259_device::ir1_w));
 
-	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
-	screen.set_refresh_hz(60);
-	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500)); // not accurate
-	screen.set_size(320, 240);
-	screen.set_visarea(0, 319, 0, 239);
-	screen.set_screen_update(FUNC(juku_state::screen_update));
+	// Электроника МС 6105.4 (60 Hz)
+	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
+	m_screen->set_refresh_hz(60);
+	m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(2500)); // not accurate
+	m_screen_x = 320, m_screen_y = 240;
+	m_screen->set_size(m_screen_x, m_screen_y);
+	m_screen->set_visarea(0, m_screen_x-1, 0, m_screen_y-1);
+	m_screen->set_screen_update(FUNC(juku_state::screen_update));
+
+	SPEAKER(config, "mono").front_center();
+	SPEAKER_SOUND(config, m_speaker).add_route(ALL_OUTPUTS, "mono", 1);
+	m_speaker->set_levels(3, speaker_levels);
 
 	TTL74148(config, m_key_encoder, 0);
 
-	KR1818VG93(config, m_fdc, 1000000);
-//  m_fdc->intrq_wr_callback().set(FUNC(juku_state::fdc_intrq_w));
+	// КР1818ВГ93
+	KR1818VG93(config, m_fdc, 1_MHz_XTAL);
```

Comment:
Is there really a 1 MHz XTAL or is the value divided down from another value?

Reply:
I think it's divided from the main 16MHz XTAL on the board. Corrected.